actionpack/lib/system_testing/driver_adapters/capybara_driver.rb

```diff
@@ -0,0 +1,86 @@
+require "system_testing/driver_adapters/web_server"
+
+module SystemTesting
+  module DriverAdapters
+    # == CapybaraDriver for System Testing
+    #
+    # The <tt>CapybaraDriver</tt> is a shim that sits between Rails and
+    # Capybara.
+    #
+    # The drivers Capybara supports are: +:rack_test+, +:selenium+, +:webkit+,
+    # and +:poltergeist+.
+    #
+    # Rails provides it's own defaults for Capybara with the Selenium driver
+    # through <tt>RailsSeleniumDriver</tt>, but allows users to use Selenium
+    # directly.
+    #
+    # To set your system tests to use one of Capybara's default drivers add
+    # the following to yur Rails' configuration test environment:
```

Comment:
"to use one of Capybara's default drivers add" -> "to use one of Capybara's default drivers, add"

line 18: yur -> your